app/Http/Controllers/WorkingDayController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StoringDoctorWorkingDayRequest;
+use App\Services\DoctorService;
+use App\Transformers\CreatedResource;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class WorkingDayController extends Controller
+{
+    protected $doctorService;
+
+    public function __construct(DoctorService $doctorService)
+    {
+        $this->doctorService = $doctorService;
+    }
+    public function store(StoringDoctorWorkingDayRequest $request)
+    {
+        $workingDay = $this->doctorService->addWorkingDay($request->except('is_all_day'));
```

Comment:
Allow to send `is_all_day` for a specific day

Reply:
done